### PR TITLE
Account page #53

### PR DESCRIPTION
**Pull request by JHFellows**

### CODE REVIEW

File: app/containers/UserContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import User from '../components/User'
+import { connect } from 'react-redux'
+
+// we'll also want to add user's own orders to state, when we have orders on state
+const mapStateToProps = (state) => ({
```
Comment (cleverettgirl) [inline app/containers/UserContainer.jsx:5]: const mapStateToProps = ({user}) => ({selectedUser: user.selectedUser })

File: app/reducers/user-reducer.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return initialUsersState
+  }
+}
+
+/* ------------------ dispatchers ------------------- */
+
+export const fetchUsers = () => dispatch => {
+  axios.get('/api/users')
+  .then(res => {
+    dispatch(get(res.data))
```
Comment (cleverettgirl) [inline app/reducers/user-reducer.jsx:37]: Not the most semantically meaningful.

File: app/reducers/user-reducer.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  })
+  .catch(err => console.error('Error fetching users :(', err))
+}
+
+export const fetchUser = (id) => dispatch => {
+  axios.get(`/api/users/${id}`)
+  .then(res => dispatch(select(res.data)))
+  .catch(err => console.error('Error fetching user :(', err))
```
Comment (cleverettgirl) [inline app/reducers/user-reducer.jsx:45]: Handle on the front --> show user there was an error at some point.

**Review by melaniemohn — APPROVED** [review]
we still want some error handling re: users on the front end, but we'll figure it out

**Pull request merged**